torchvision/prototype/transforms/_auto_augment.py

```diff
@@ -395,4 +444,113 @@ def forward(self, *inputs: Any) -> Any:
         else:
             magnitude = 0.0
 
-        return self._apply_transform(sample, transform_id, magnitude)
+        return _put_into_sample(
+            sample,
+            id,
+            self._apply_image_transform(sample, transform_id, magnitude, interpolation=self.interpolation, fill=fill),
```

Comment:
Nit: Could you move this to a separate line? aka

sample = self._apply_image_transform(sample, ...)